PLT-977 Create parameter store and secrets manager migration scripts

jscott-nava opened the pull request:
## 🎫 Ticket

https://jira.cms.gov/browse/PLT-977

## 🛠 Changes

- created a new 'tools' directory in the platform repo.
- added migrate_parameters and migrate_secrets Python scripts.
- added a settings.ini file to contain required configuration info.
- added a README with basic usage instructions.

## ℹ️ Context

This PR is intended to preserve the scripts that were developed for the purpose of migrating parameters and secrets from the legacy BCDA and AB2D AWS accounts to the Greenfield prod and non-prod accounts.

While this PR was opened against the Platform repo, it is not clear if this is the desired repo for an internal tools directory. If a repo such as cdap-private is more appropriate for tools then this PR can be recreated in that repo.

## 🧪 Validation

These scripts were tested with a series of dry-runs followed by some test migrations of very limited subsets of parameters and secrets (i.e. > 3). 

The full migration has now been run, and further validation in the form of spot-checking various parameters and secrets was conducted.

gsf reviewed (commented):
It was raised that the inclusion of credentials in the ini file could lead to accidentally pushing those credentials to the repo. Could environment variables be used for those credentials instead? That will also make this more compatible with the short term access key environment variables copied from cloudtamer.

jscott-nava commented:
@gsf Please review the following updates based on your feedback:

- removed AWS creds from the settings.ini and now get them from the environment; updated the README with specific commands to fill in and run.
- merged the migrate_parameters and migrate_secrets scripts into one migrate_items script and removed any duplicate code; add a 'RESOURCE-TYPE' variable to the settings.ini that must be either 'parameter' or 'secret'.
- moved the directory containing the script, settings and README files under the existing scripts directory.

gsf reviewed (approved):
Looks good! Thanks for addressing all comments
